cmd/cli/tracer.go

```diff
@@ -21,7 +21,9 @@ func initTracerCommand() *cobra.Command {
 	tracerCMD.Flags().Bool("allow-local-ranges", true, "allows access to local IP ranges")
 	tracerCMD.Flags().Bool("allow-github-meta", false, "allows access to GitHub meta IP ranges (https://api.github.com/meta)")
 	tracerCMD.Flags().String("allowed-hosts", "", "enter allowed hostnames (example.com, .github.com)")
+	tracerCMD.MarkFlagRequired("allowed-hosts")
```

Comment:
Should not be marked as required.
The tool can be run in monitor mode.
The allowed IP might have passed.